docs/section10/10-6.md

```diff
@@ -0,0 +1,35 @@
+# Airside systems – Rule 10-6
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False   
+**Rule ID:** 10-6
+ 
+**Rule Description:** For HVAC systems designed, mechanical cooling equipment efficiencies shall be adjusted to remove the supply fan energy from the efficiency rating.  
+**Rule Assertion:** P-RMR = expected value                                            
+**Appendix G Section:** Table G3.1 Section 10 b.  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each CoolingSystem data table  
+
+**Applicability Checks:** 
+
+1. HVAC with mechanical cooling.
+
+**Function Call:** None
+
+**Manual Check:** 
+
+ 1. Check that the mechanical equipment cooling efficiency has been adjusted to remove the supply fan energy from the efficiency rating. 
+
```

Comment:
I don't think this RDS, as it stands now, is doing much. I posted RMR issues about adding COPnfcooling and COPnfheating to the schema. Then, this RDS could check that these data elements are included for each packaged HVAC system in the proposed design. We could perhaps also include a check to confirm that COPnfcooling is not just a unit convertion for EER.

Reply:
@mkarpman i do not know how this would work. E+ only takes in the COPnfcooling and heating values, there is no way to enter rated EER etc.
Even if we add COPnfcooling/COPnfheating, we would need to assume that values for EER etc might not be provided and this check cannot be performed. How does eQuest handle it?

I had a similar comment for verifying boiler efficiency, it is unlikely that the RMR would include AFUE, Et, Ec for models generated using E+, E+ only takes in Et. Again, not sure how eQuest handles this.

Reply:
@claperle and @mkarpman not sure where we landed with MK's first comment. Looks like we're now checking if full load eff and non fan eff is entered but are not doing a unit conversion check. I think a check just on no fan eff is sufficient but want to amke sure we're not missing anything